transforms/pure-component.js

```diff
@@ -0,0 +1,60 @@
+const REACT_COMPONENT_CLASS_DEFINITION = {
+  superClass: {
+    type: 'MemberExpression',
+    object: {
+      name: 'React'
+    },
+    property: {
+      name: 'Component'
+    }
+  }
+};
+
+function isRenderMethod(node) {
+  return (
+    node.type == 'MethodDefinition' &&
+    node.key.type == 'Identifier' &&
+    node.key.name == 'render'
+  );
+}
+
+const THIS_PROPS = {
+  object: {
+    type: 'ThisExpression'
+  },
+  property: {
+    name: 'props'
+  }
+};
+
+function replaceThisProps(node) {
+  return j(node)
```

Comment:
I started refactoring and then tried to write tests and the tests didn't work and I didn't have time to look into it earlier.